src/main/scala/org/apache/spark/sql/sparkdl_stubs/UDFUtils.scala

```diff
@@ -16,9 +16,10 @@
 package org.apache.spark.sql.sparkdl_stubs
 
 import java.util.ArrayList
-import scala.collection.JavaConverters._
 
+import scala.collection.JavaConverters._
 import org.apache.spark.internal.Logging
+import org.apache.spark.sql.catalyst.FunctionIdentifier
```

Comment:
should be below the next line. did eclipse put these in this ordering?

Reply:
Yeah Idea did. I though I set it up with the guidelines.